tests/test_invites.py

```diff
@@ -90,6 +90,27 @@ def test_invited_email_address_obfuscation(self):
 
         self.assertEqual(redacted_address, "...@1...")
 
+        # Try using a username separator string
+        self.sydent.third_party_invite_username_separator_string = "-"
+
+        email_address = "johnathon-jingle-smithington@company-town.notarealtld"
+        redacted_address = store_invite_servlet.redact_email_address(email_address)
+        # Each individual component of the username should be obfuscated, but not the domain
+        self.assertEqual(redacted_address, "johnat...-jin...-smithi...@company-...")
```

Comment:
Shouldn't this be `@compan...`? This seems to show it also splitting the domain on `-`?

Reply:
`email.third_party_invite_domain_obfuscate_characters` is set to 8 for this test, so it should split there. But you're right, splitting on `-` is quite confusing.

Reply:
Ah, it just happens to be that that's 8 characters. 🤦 Maybe we should just change the domain count for this test so that it doesn't seem to be doing the wrong thing.